CMakeLists.txt

```diff
@@ -67,23 +67,32 @@ execute_process(
   OUTPUT_STRIP_TRAILING_WHITESPACE
 )
 
+
 set(CLANG_CXXFLAGS "")
 set(CLANG_INCLUDEDIR "${LLVM_SRC}/tools/clang/include" "${LLVM_BUILD}/tools/clang/include")
+
+set(ALIVE_DIR "${CMAKE_SOURCE_DIR}/third_party/alive2")
+set(ALIVE_LDFLAGS "-L ${ALIVE_DIR}/build -lalive2")
+
+set(CLANG_CXXFLAGS "-I\"${LLVM_SRC}/tools/clang/include\" -I\"${LLVM_BUILD}/tools/clang/include\"")
```

Comment:
That is already specified by `CLANG_INCLUDEDIR`.

Reply:
Thanks for pointing it out, I think I messed up rebasing at some point.

Reply:
Updated